pandas/io/excel.py

```diff
@@ -68,40 +67,43 @@
     * 1 -> 2nd sheet as a DataFrame
     * "Sheet1" -> 1st sheet as a DataFrame
     * [0,1,"Sheet5"] -> 1st, 2nd & 5th sheet as a dictionary of DataFrames
-    * None -> All sheets as a dictionary of DataFrames
-
-sheetname : string, int, mixed list of strings/ints, or None, default 0
+    * None -> All sheets as a dictionary of DataFrames.
```

Comment:
I'd use `:` instead of `->` (it's how we have lists in parameters everywhere)

I'd merge the first 3 sentences of the description in a single paragraph, and remove the part of what is being returned (that will be explained in the `Returns` section).

Ad colon after `Available cases:` (and lowercase `C`)